docs/content/templates/single-page-form/SinglePageForm.tsx

```diff
@@ -50,11 +50,11 @@ const formSchema = yup
 			.object({
 				from: yupDateField
 					.required('Enter a valid date')
-					// Ensures the start date is always after the end date
+					// Ensures the end date is always after the start date
```

Comment:
I don't mind too much that this comment is identical to the one below (because it's still true), but my suggestion was for "Ensures the start date is always after the end date" so you can see at a glance that this is for the start date